docs/releasenotes/release-1.121.md

```diff
@@ -1,26 +1,23 @@
-# v1.120.0
+# v1.121.0
 
-** This version is not yet released; this document is gathering release notes for the future release **
+* Special shout-outs to @600lyy, @acpana, @barney-s, @gemmahou, @haiyanmeng, @hankfreund, @jasonvigil, @jiefenghe, @jingyih, @justinsb, @maqiuyujoyce, @ostrain, @xiaoweim, @yuwenma, @ziyue-101 for their contributions to this release.
 
-* ...
+## Announcement 
 
-* Special shout-outs to ... for their
-  contributions to this release.
-TODO: list contributors with `git log v1.121.0... | grep Merge | grep from | awk '{print $6}' | cut -d '/' -f 1 | sort | uniq`
+* The `state-into-spec` is default to `Absent` in any *new* ConfigController clusters. 
+We plan to apply this default to *all* the ConfigController clusters in v1.122 and then
+to all ConfigConnector clusters in v1.123.   
 
-## Resources promoted from alpha to beta:
+## Direct Cloud Reconciler:
 
-*When resources are promoted from alpha to beta, we (generally) ensure they follow our best practices: use of refs on fields where appropriate,
-output fields from GCP APIs are in `status.observedState.*`
-
-* `PlaceholderKind`
-
-## New Resources:
-
-* Added support for `PlaceholderKind` (v1beta1) resource.
+* `DataformRepository`
```

Comment:
Maybe include " (alpha)" - I assume it's alpha

Reply:
oh, good point!